chef_master/source/dsl_recipe.rst

```diff
@@ -1555,33 +1554,21 @@ The following Microsoft Windows platform-specific helpers can be used in recipes
    * - Helper
      - Description
    * - ``cluster?``
-     - Use to test for a Cluster SKU (Windows Server 2003 and later).
+     - Use to test for a Cluster SKU (Windows Server 2012 and later).
    * - ``core?``
-     - Use to test for a Core SKU (Windows Server 2003 and later).
+     - Use to test for a Core SKU (Windows Server 2012 and later).
    * - ``datacenter?``
      - Use to test for a Datacenter SKU.
    * - ``marketing_name``
      - Use to display the marketing name for a Microsoft Windows platform.
-   * - ``windows_7?``
-     - Use to test for Windows 7.
    * - ``windows_8?``
      - Use to test for Windows 8.
    * - ``windows_8_1?``
      - Use to test for Windows 8.1.
    * - ``windows_10?``
      - Use to test for Windows 10.
-   * - ``windows_2000?``
-     - Use to test for Windows 2000.
    * - ``windows_home_server?``
      - Use to test for Windows Home Server.
-   * - ``windows_server_2003?``
-     - Use to test for Windows Server 2003.
-   * - ``windows_server_2003_r2?``
-     - Use to test for Windows Server 2003 R2.
-   * - ``windows_server_2008?``
-     - Use to test for Windows Server 2008.
-   * - ``windows_server_2008_r2?``
-     - Use to test for Windows Server 2008 R2.
```

Comment:
I have mixed feelings on this. These are all still in the windows cookbook. We're in the process of deprecating all of them, but for the moment they're all still valid. That being said no one should be using the old xp helper today so maybe it makes sense to just kill it all